src/test/java/com/example/kspot/KspotApplicationTests.java

```diff
@@ -14,18 +17,45 @@ class KspotApplicationTests {
   private JdbcTemplate jdbcTemplate;
 
   @Test
-  void printAllTables() {
-    String[] tables = {"locations", "contents"};
+  void checkContentsTable(){
+    String sql = "SELECT content_id, category, title, poster_image_url, release_date FROM contents";
+    List<Map<String, Object>> results = jdbcTemplate.queryForList(sql);
+
+    assertThat(results).isNotEmpty();
+
+    boolean contents = results.stream()
+        .anyMatch(row -> row.get("content_id").toString().equals("7")
+        && "기생충".equals(row.get("title"))
+        && "2019-05-30".equals(row.get("release_date").toString()));
+    assertThat(contents).isTrue();
+  }
```

Comment:
```suggestion
void checkTableStructure() {
    // 테이블 존재 여부 확인
    String sql = "SELECT COUNT(*) FROM information_schema.tables WHERE table_name = 'contents'";
    Integer count = jdbcTemplate.queryForObject(sql, Integer.class);
    assertThat(count).isEqualTo(1);
    
    // 필수 컬럼 존재 여부 확인
    String columnSql = "SELECT column_name FROM information_schema.columns WHERE table_name = 'contents'";
    List<String> columns = jdbcTemplate.queryForList(columnSql, String.class);
    assertThat(columns).contains("content_id", "title", "release_date");
}
```

이런식으로 테이블이 존재하는지 검증하는게 좋을것 같아요 :)